### ISSUE TITLE
v2.9.5 doesn't create Actors anymore

### ISSUE THREAD

**Issue by Jedrzej94** [samp-incognito/samp-streamer-plugin#388]
No idea what should I add here, specify what would you like to dive into within my code and I will provide.

v2.9.4 creates actors all right, while v2.9.5 simply doesn't. At releases page I noticed there are some actors changes (provided by ziggi), so I would probably look deeper into that.

Let me know what you need from my side to know.

```
hook OnGameModeInit()
{
    for(new i = 1; i != sizeof(eActorsData); i++)
    {
        CreateDynamicActor(
            eActorsData[i][e_iActorModelId], 
            eActorsData[i][e_fActorX], 
            eActorsData[i][e_fActorY], 
            eActorsData[i][e_fActorZ], 
            eActorsData[i][e_fActorAng],
            .streamdistance = (STREAMER_ACTOR_SD / 2.0),
            .priority = eActorsData[i][e_iPriority]
        );

        eActorInfo[i][e_iChatBubbleLabelId] = INVALID_3DTEXT_ID;
    }
    return 1;
}
```

**Comment by djacicd** [comment]
You need to assign them virtual world in order to show them... It's probably bug, they should be visible in all virtual worlds if you don't assign them specific one.

**Comment by Jedrzej94** [comment]
Weirdly enough, that worked. Leaving the issue open, might be worth investigating further, since the param is optional by default.

**Comment by Jedrzej94** [comment]
Even worse. 2.9.5 doesn't detect NPCs entering dynamic areas anymore. I have to roll back to 2.9.4 - seems like the latest version wasn't tested that much?

**Comment by samp-incognito** [comment]
Yes, ziggi's changes were meant to address the problems with actors and pickups in #316, but apparently setting the virtual world to `-1` no longer works, if I'm understanding this issue correctly. Can anyone confirm if this is limited to actors, or if pickups are affected as well?

**Comment by ziggi** [comment]
Actors with -1 as virtual world is not working because SA-MP isn't support this. But pickups supports it and works well (tested on production). But I think what `CreateDynamicActor` should use virtual world `0` by default.
